docs/ha-considerations.md

```diff
@@ -83,23 +83,32 @@ There are some placeholders in `bash` variable style to fill in:
 
 The above `keepalived` configuration uses a health check script `/etc/keepalived/check_apiserver.sh` responsible for making sure that on the node holding the virtual IP the API Server is available. This script could look like this:
 
-```
+```bash
 #!/bin/sh
 
 errorExit() {
     echo "*** $*" 1>&2
     exit 1
 }
 
-curl --silent --max-time 2 --insecure https://localhost:${APISERVER_DEST_PORT}/ -o /dev/null || errorExit "Error GET https://localhost:${APISERVER_DEST_PORT}/"
-if ip addr | grep -q ${APISERVER_VIP}; then
-    curl --silent --max-time 2 --insecure https://${APISERVER_VIP}:${APISERVER_DEST_PORT}/ -o /dev/null || errorExit "Error GET https://${APISERVER_VIP}:${APISERVER_DEST_PORT}/"
+# replace this with the virtual IP address negotiated between the `keepalived` cluster hosts
+APISERVER_VIP=${APISERVER_VIP_VALUE}
+# replace this with the port through which Kubernetes will talk to the API Server
+APISERVER_DEST_PORT=${APISERVER_DEST_PORT_VALUE}
+
+url="https://localhost:$APISERVER_DEST_PORT"
```

Comment:
uppercase URL

Reply:
Does kubeadm has a different code style for bash script? Normally local non constant vars should be lower case.

Reply:
what is the rest of the doc doing?

Reply:
I don't see any other scripts in this doc. If the naming practice is different than the normal ones, just tell me so, or better, put code style guideline somewhere instead of expect people to go through all the docs and find random scripts to see what is the "consistent" style.

Reply:
there is no convention documented in this repo or kubernetes/kubernetes, but uppercase even for local vars seems more common.

Reply:
Almost all the shell code styles I saw use lowercase in this case, including [Google's shell style guide](https://google.github.io/styleguide/shellguide.html#variable-names)